libtransact/src/contract/archive/mod.rs

```diff
@@ -325,6 +345,25 @@ mod tests {
         assert!(find_scar("nonexistent", "0.1.0", &[&dir]).is_err());
     }
 
+    /// Verify that an error is returned when the contract name contains '_'.
+    #[test]
+    fn find_scar_invalid_name() {
+        let dir = new_temp_dir();
+
+        assert!(find_scar("non_existent", "0.1.0", &[&dir]).is_err());
+    }
+
+    /// Verify that an error is returned when the contract name contains '_'.
+    #[test]
+    fn find_scar_invalid_semver() {
+        let dir = new_temp_dir();
+
+        let err = find_scar("nonexistent", "0..1..0", &[&dir]);
+
+        assert!(err.is_err());
+        assert!(find_scar("nonexistent", "0..1..0", &[&dir]).is_err());
```

Comment:
Looks like the same thing is being tested twice.

Reply:
Whoops, yeah, forgot to update that test